src/Tests/dotnet-run.Tests/GivenDotnetRunRunsCsProj.cs

```diff
@@ -635,7 +635,7 @@ public void ItForwardsEmptyArgumentsToTheApp()
                 .Should()
                 .Pass()
                 .And
-                .HaveStdOutContaining($"0 = a{Environment.NewLine}1 = {Environment.NewLine}2 = c");
+                .HaveStdOutContaining($"0 = a{Environment.NewLine}1 = c");
```

Comment:
explicitly calling out this test change as well: S.CL strips empty arguments now, as I understand it this is in keeping with shell argument parsing semantics across multiple platforms: https://github.com/jonsequitur/command-line-api/commit/260a559f5a5f4ec8d075a7accec1f2a185528a43